nextjs-app/shared/components/EnhancedProjectCard/EnhancedProjectCard.tsx

```
@@ -164,78 +159,42 @@ export function EnhancedProjectCard({
           </span>
         )}
 
         {/* Title */}
-        <h3
-          className={cn(
-            "font-display font-semibold text-foreground",
-            "text-base tablet:text-lg",
-            "leading-tight"
-          )}
-        >
+        <h3 className={styles.title}>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)